### ISSUE TITLE
Offline images not working

### ISSUE THREAD

**Issue by Alkarex** [Alkarex/EasyRSS#8]
As reported by @Alwaysin in https://github.com/Alkarex/EasyRSS/issues/5#issuecomment-50785812

**Comment by Alwaysin** [comment]
As of today, the bug still exists :(

**Comment by Alkarex** [comment]
Yes, I have not looked at it yet :-/ My last work was to add TLSv1.2 and to start the F-Droid process.

**Comment by matolivier** [comment]
Hello Alexandre.

Did you managed to find a time in order to dig this issue? I see no pictures on articles whatever settings I have. But I still can click on them to see it.

![screenshot_20160824-080053](https://cloud.githubusercontent.com/assets/18326674/17925365/f76b55a4-69ec-11e6-9f1e-a59bdc59838b.png)

![screenshot_20160824-080103](https://cloud.githubusercontent.com/assets/18326674/17925367/fad763a4-69ec-11e6-926f-f929e348854d.png)

![screenshot_20160824-080112](https://cloud.githubusercontent.com/assets/18326674/17925372/fe5ce404-69ec-11e6-8b75-66620ea7dfea.png)
